src/components/AvatarGroup/Avatars.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledAvatar, StyledImage, StyledInitialsText } from './styled';
+import { Avatar } from './types';
+import RenderIf from '../RenderIf';
+
+interface Props {
+    maxAvatars?: number;
+    size?: string;
+    avatars?: Avatar[];
+}
+
+const Avatars: React.FC<Props> = props => {
+    const { avatars = [], maxAvatars = 3, size } = props;
+    const items = avatars.slice(0, maxAvatars).map((avatar, idx) => {
+        const { src, initials, icon } = avatar;
+
+        const key = `avatar-${idx}`;
+        const zIndex = maxAvatars - idx;
+        return (
+            <StyledAvatar
+                key={key}
+                zIndex={zIndex}
+                firstChild={idx === 0}
```

Comment:
let's try to not mix js with jsx to maintain separation of concerns and instead use:
```const isFirstChild = index === 0;